UI/html_files/signup.html

```diff
@@ -51,24 +51,24 @@ <h2><b>Motto:</b> Together we stand divided we fall</h2>
         <div class="logo"><img src="../images/logo1.png" alt="logo"></div>
         <p class="label">Signup</p>
         <div class="textFields">
-          <div class="text1"><input type="text" id="clients" placeholder="First Name"></div>
-          <div class="text1"><input type="password" id="passAdmin" placeholder="Last Name" value=""></div>
+          <div class="text1"><input type="text" id="clients" placeholder="First Name" required></div>
+          <div class="text1"><input type="password" id="passAdmin" placeholder="Last Name" value="" required></div>
           <div class="text1"><input type="text" id="clients" placeholder="Email"></div>
-          <div class="text1"><input type="password" id="passAdmin" placeholder="Password" value=""></div>
-          <div class="text1"><input type="text" id="clients" placeholder="Role"></div>
+          <div class="text1"><input type="password" id="passAdmin" placeholder="Password" value="" required></div>
+          <div class="text1"><input type="text" id="clients" placeholder="Role" required></div>
         </div>
         <div id="btn2"></div>
         <div id="btn">
           <!-- <button class="btn_post" onclick="checkClient()"> User</button> -->
-          <button class="btn_post"><a href="homepage.html"
-              style="color:white;font-size: 18px;">Login</a> </button>
+          <button class="btn_post" onclick= "window.location.href ='homepage.html';">
+            <p style="color:white;font-size: 16px;">Login</p>
```

Comment:
You need to get rid of inline styling... Have all the CSS in the CSS file.

Reply:
already done, Please check!